qpc/messages.py

```diff
@@ -232,7 +233,9 @@
 BECOME_PASSWORD = 'Provide a privilege escalation password to be used when '\
     'running a network scan.'
 
-
+JOB_ID_NOT_FOUND = 'Error finding id %s.'
+JOB_ID_STATUS = 'Job id: %s status is %s'
```

Comment:
This message would produce something like: 
```Job id: 2 status is Job is complete.```
so I personally think that we should take out this message altogether and just print the status message inside the json data. Ie. "Job is complete." I think this would be fine because we just requested the status so we don't need to repeat the job id.

Reply:
I think that since it is async now, if you run the command more than once. It helps to have the id while looking at the logs.

Reply:
True

Reply:
Changed to ```status``` instead of ```status_message```